src/please.go

```diff
@@ -367,6 +368,12 @@ var opts struct {
 			} `positional-args:"true"`
 		} `command:"filter" description:"Filter the given set of targets according to some rules"`
 	} `command:"query" description:"Queries information about the build graph"`
+	Codegen struct {
+		Gitignore string `long:"update_gitignore" description:"The gitignore file to write the generated sources to"`
+		Args   struct {
+			Targets []core.BuildLabel `positional-arg-name:"targets" description:"Targets to filter"`
+		} `positional-args:"true"`
+	} `command:"codegen" description:"Builds all code generation targets in the repository and prints the generated files."`
```

Comment:
I sort of feel like this should really be `plz gencode` so it's a verb but `codegen` is more common...

Reply:
Yeah good point. We could steal go's nomenclature: `go generate`